src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -134,6 +136,20 @@ def frequency(self) -> GridFrequency:
 
         return self._frequency_instance
 
+    def voltage(self) -> VoltageStreaming:
```

Comment:
I'm fine with merging this as is, but for me `frequency()` and `voltage()` is starting to look to me as they should be part of a logical meter object. This discussion is part of:

* https://github.com/frequenz-floss/frequenz-sdk-python/discussions/782

And probably this won't be addressed in 1.0.

Reply:
We've agreed on keeping `voltage()` at the top level for SDK v1